docs/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can dispense with the whole content of /docs for now as we are in the process of determining a SmallRye wide docs solution

Reply:
Done